saleor/settings.py

```diff
@@ -1005,14 +1005,26 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 BREAKER_BOARD_FAILURE_THRESHOLD_PERCENTAGE = int(
     os.environ.get("BREAKER_BOARD_FAILURE_THRESHOLD_PERCENTAGE", 50)
 )
+BREAKER_BOARD_FAILURE_THRESHOLD_PERCENTAGE_HALF_OPEN = int(
```

Comment:
Consider renaming `HALF-OPEN` to `RECOVERY` here.

e.g. `BREAKER_BOARD_RECOVERY_FAILURE_THRESHOLD_PERCENTAGE`